src/LiveComponent/doc/index.rst

```diff
@@ -3715,6 +3715,11 @@ Debugging Components
 Need to list or debug some component issues.
 The `Twig Component debug command`_ can help you.
 
+To debug your PHP code, you can inject a `private LoggerInterface $logger` in your `__construct`.
+Then you can use `$this->logger->debug('Message', ['var' => $var]);` and see the output in `var/log/dev.log`.
+`dump` won't display anything anywhere (not even the debug toolbar).
```

Comment:
This is not true. 

During the first render, dumps are shown either in the template or in the debug toolbar.

On following request they are shown on the profiler, like any AJAX request.